src/functionalTest/java/uk/gov/hmcts/darts/dailylist/DailylistFunctionalTest.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.hmcts.darts.dailylist;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.darts.FunctionalTest;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DailylistFunctionalTest extends FunctionalTest {
+
+    public static final String POST_DAILYLIST_URL = "/dailylists";
+
+    public static final String POST_DAILYLIST_RUN_URL = "/dailylists/run";
+
+    @Test
```

Comment:
This test needs a better assertion criteria. The the xml document will fail but the json one should be processed as we only process the json document as part of the daily list processing.